Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/Delete/DeleteStreetcodeHandler.cs

```diff
@@ -25,6 +30,12 @@ public async Task<Result<Unit>> Handle(DeleteStreetcodeCommand request, Cancella
         _repositoryWrapper.StreetcodeRepository.Delete(streetcode);
 
         var resultIsSuccess = await _repositoryWrapper.SaveChangesAsync() > 0;
-        return resultIsSuccess ? Result.Ok(Unit.Value) : Result.Fail(new Error("Failed to delete a streetcode"));
+
+        if (!resultIsSuccess)
```

Comment:
Is there any point of refactoring it in this way?
also it'd would be better to name this variable as `isSuccessResult` IMHO

Reply:
if it's common convention, it's okay with me

Reply:
but I see you've approved this PR with ternary operator in similiar place - https://github.com/ita-social-projects/StreetCode/pull/132/files#diff-a5606f5d4d156694e3b38579aa50412b5b3e30bbd53d977626ba3f8b4624b6d2R31

So are we really talking about some conventions here?